src/consensus/pbft/pbft.h

```diff
@@ -155,6 +155,7 @@ namespace pbft
       general_info.view_timeout = 5000;
       general_info.status_timeout = 100;
       general_info.recovery_timeout = 9999250000;
+      general_info.max_requests_between_signatures = 50;
```

Comment:
If I understand correctly we are relying on the this number here (`50`) to be the transaction interval between signatures, but also on the `sig_max_ms` millisecond interval which will issue signatures via history on `tick`. The latter is set in the `SignatureIntervals` struct which is consensus agnostic. So the `sig_max_tx` from that struct should be used to set the value here.

Reply:
sig_max_ms is already used via the frontend ticks in the same way raft uses it